docs/arch-overview.md

```diff
@@ -0,0 +1,82 @@
+
+The K8ssandraCluster object must be created in the control plane cluster.
+
+K8ssandra Operator is deployed in each cluster. The operator consists of several controllers. Some controllers, notably the K8ssandraCluster controller, only run in the control plane cluster. This is the main reason that the K8ssandraCluster object needs be created in the control plane cluster. 
+
+# CassandraDatacenter
+Here we discuss the `cassandra.datacenters` property which is an array. K8ssandra Operator creates a CassandraDatacenter for each element. 
+
+Notice the `name: dc1` that is highlighted in green. This tells K8ssandra Operator to create a CassandraDatacenter named `dc1`. The green arrow in the diagram points to the CassandraDatacenter object. The `size` property declares that there should be 3 Cassandra nodes.
+
+<!-- TODO: Add link to configuration doc when its available -->
+
+Cass Operator manages the CassandraDatacenter. Note the arrow from the CassandraDatacenter to the StatefulSet `test-dc1-default-sts`. Cass Operator creates the StatefulSet. The StatefulSet controller, which is part of Kubernetes itself, creates the actual pods that run Cassandra.
+
+
+## k8sContext
+K8ssandra Operator creates `dc1` in the `east` cluster. The `k8sContext` property determines in which Kubernetes cluster the operator creates the CassandraDatacenter. If not specified the operator creates the CassandraDatacenter in the control plane cluster.
+
+To understand what values can be specified for `k8sContext` we have to talk about ClientConfigs which are covered in the next section.
+
+# ClientConfig
+A ClientConfig is another custom resource provided by K8ssandra Operator; however, it is not part of a K8ssandraCluster. The operator learns about remote Kubernetes clusters from ClientConfig objects. They provide information necessary for the operator to create connections and manage objects in remote clusters.
+
+The operator identifies remote clusters using the `contextName` property. The property is optional. If not specified, the operator uses the ClientConfig name to identify the remote cluster.
+
+The arrow going from the `k8sContext` property to the ClientConfig object which has a `contextName` property represents an indirect relationship. Creating and managing ClientConfigs is separate and independent from managing K8ssandraClusters.
+
+**Note:** See [Remote Cluster Connection Management][remote-k8s-access/README.md] for more information about ClientConfigs. 
+
+# Telemetry
+Next we look at the `cassandra.telemetry` property. When it is enabled, K8ssandra Operator creates a ServiceMonitor for each CassandraDatacenter.
+
+**Note:** The ServiceMonitor custom resource is provided by [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator).
```

Comment:
Probably worth a mention of why the user should care about that fact... that the CRDs need to be installed for this to work.